cranelift/filetests/filetests/isa/x86/simd-load-binemit.clif

```diff
@@ -5,6 +5,7 @@ target x86_64 skylake
 function %load_extend() {
 block0:
     [-,%rdx]     v1 = iconst.i64 0x0123_4567_89ab_cdef
-    [-,%xmm1]    v2 = uload8x8 v1  ; bin: heap_oob 66 40 0f 38 30 0a
+    [-,%xmm1]    v3 = uload8x8 v1+0  ; bin: heap_oob 66 40 0f 38 30 0a
+    [-,%xmm2]    v4 = uload8x8 v1+20  ; bin: heap_oob 66 40 0f 38 30 0a
```

Comment:
@bnjbvr, this is the mystery I mentioned in Zulip: Cranelift can't find an encoding for the second instruction, the one with the `+20` offset.

Reply:
Figured this out: the `uload*` instructions need to have encodings not just for the `fld` recipe but also `fldDisp8` and `fldDisp32`.